### PR TITLE
Develop labyrinth generator

### PR DESCRIPTION
**Pull request by northfieldzz**

### CODE REVIEW

File: Assets/Cube.prefab [hunk 1]
```diff
@@ -0,0 +1,95 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!1 &2141419430373024782
+GameObject:
+  m_ObjectHideFlags: 0
```
Comment (sho-rrr) [inline Assets/Cube.prefab:5]: aiueo

**Review by sho-rrr — COMMENTED** [review]
OK

File: Assets/Cube.prefab (hunk 1 above)
Comment (northfieldzz) [reply Assets/Cube.prefab:5]: unko

**Review by sho-rrr — APPROVED** [review]
ok

**Comment by sho-rrr** [comment]
merge

**Pull request merged**